src/test/scala/mesosphere/marathon/api/SystemResourceTest.scala

```diff
@@ -95,17 +94,30 @@ class SystemResourceTest extends AkkaUnitTest with JerseyTest {
       Option(response.getMetadata().getFirst("Content-type")).value.toString should be("text/html")
     }
 
-    "Get metrics" in new Fixture {
-      When("The metrics are requested")
+    "get metrics" in new Fixture {
+      When("the metrics are requested")
       val response = resource.metrics(auth.request)
 
-      Then("The metrics are sent")
+      Then("the metrics are sent")
+      response.getStatus shouldBe 200
+      Option(response.getMetadata.getFirst("Content-Type")).value.toString should be("application/json")
+
       val metricsJson = Json.parse(response.getEntity.asInstanceOf[String])
-      metricsJson \ "start" shouldBe a[JsDefined]
```

Comment:
shouldn't we have tests for both kamon and dropwizard until we drop the old one?